### PR TITLE
Seeded key pair tests

### PR DESCRIPTION
**Pull request by laszlourszuly**

### CODE REVIEW

File: src/modules/identity/RadixSimpleIdentity.spec.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        const identity2 = manager.generateSimpleIdentityFromSeed(Buffer.from(seed))
+
+        await identity1.signAtom(atom1)
+        await identity2.signAtom(atom2)
+
+        const signature1 = atom1.signatures[identity1.address.getUID().toString()]
+        const signature2 = atom2.signatures[identity2.address.getUID().toString()]
+
+        expect(signature2.r.toString()).is.equal(signature1.r.toString())
+        expect(signature2.s.toString()).is.equal(signature1.s.toString())
```
Comment (laszlourszuly) [inline src/modules/identity/RadixSimpleIdentity.spec.ts]: This feels like a naive approach to me. Surely there must be a better way to verify signature. Any comments @MuncleUscles

Comment (MuncleUscles) [reply src/modules/identity/RadixSimpleIdentity.spec.ts]: You should be able to do `expect(signature1).to.deep.equal(signature2)`, same elsewhere

But yes, a more valuable test would be to check if the signature verifies properly. 
RadixAddress has a .verify method, if you pass it `atom.getHash()` and the signature it should succeed

File: src/modules/account/RadixAccount.spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai'
+import 'mocha'
+import { RadixAccount } from '../..'
+
+describe('RadixAccount', () => {
```
Comment (MuncleUscles) [inline src/modules/account/RadixAccount.spec.ts]: If you're also doing this in java, would be good to test one seed and output pair from there, to check that we get the same result in both libraries.

File: src/modules/identity/RadixSimpleIdentity.spec.ts (hunk 1 above)
Comment (laszlourszuly) [reply src/modules/identity/RadixSimpleIdentity.spec.ts]: Sweet! The "deep equals" did the trick!

The `verify` was also valuable, but there seems to be some issues with the current implementation. I have proposed a suggested solution in another PR. Let's continue this discussion there.

**Pull request merged**